### PR TITLE
Miniaudio WASAPI loopback

### PR DESCRIPTION
**Pull request by s-ol**
Implementation for #64 😃

### CODE REVIEW

File: src/ui/PlayerOverlay.cpp [hunk 1]
```diff
@@ -376,7 +376,7 @@ void PlayerOverlay::buttonPressed( const void * sender, ofTouchEventArgs & args
 
 void PlayerOverlay::inputSelected(const void * sender, FMenu<string>::Option & opt) {
 	auto menu = opt.button->findParentOfType<FMenu<string>>();
-	auto config_ref = opt.button->getProperty<ma_device_info>("ma_device_info"); 
+  auto config_ref = opt.button->getProperty<ma_device_info>("ma_device_info");
```
Comment (s-ol) [inline src/ui/PlayerOverlay.cpp]: ah, my VS config is all screwed up, I'll have to fix that...

**Comment by kritzikratzi** [comment]
this is really great, and it gives so much better results than the "stereo mix" devices that can be enabled in windows. thanks! happily accepting it in, you should really consider proposing it to miniaudio itself. imho it's a great addition.

**Comment by kritzikratzi** [comment]
oh, i was a bit too quick to accept this. 

output device selection doesn't work anymore, also now i get crashes when unplugging/removing a  device while in use. 


a few things that might happen: 
* ma_device_reinit__wasapi seems to not consider the new streamFlagsIn
* same for init of playback devices


looking into it, i'll post back if i find more

**Comment by kritzikratzi** [comment]
ok, i could temporarily fix it by CHANGING

    struct{ .... 
        DWORD streamFlagsIn; 
    }

to 

    struct{...
        DWORD streamFlagsIn = 0; 
    }


in this commit: https://github.com/kritzikratzi/Oscilloscope/commit/9fa648ea4afbf3fc32a54c40908cb02ebdf4ffdd#diff-05224fa8611d9a6de1aac2d7c2b9f099


do you have the muse to properly fix this? (i'm guessing it should sometimes be set to loopback in some *_reinit_* places... )

**Comment by s-ol** [comment]
I'll take a look soon, but it might take until next week.

**Pull request merged**